registration/views.py

```diff
@@ -516,7 +516,18 @@ def done(self, form_list, **kwargs):
         if self.request.session[SESSION_REGISTRATION_UPDATE]:
             return django.shortcuts.redirect('update_done')
         else:
-            return django.shortcuts.redirect('email_next_of_kin')
+            return self.get_next_of_kin_page()
+
+    def get_next_of_kin_page(self):
+        try:
+            count = cache.incr('done_registration_count')
+        except ValueError:
+            count = 0
+            cache.set('done_registration_count', 0)
+        if count % 2:
+            return django.shortcuts.redirect('email_next_of_kin_1')
+        else:
+            return django.shortcuts.redirect('email_next_of_kin_2')
```

Comment:
@danaspiegel What do you think about this approach for selecting which URL to redirect to?